snippets/crowdstrike-falcon-actors/crowdstrike-falcon-actors.storm

```diff
@@ -0,0 +1,116 @@
+/*
+Name: crowdstrike-falcon-actors.storm
+Original Author: bartosz.roszewski@smartcontract.com
+Modified By: <>
+Date: 2023-10-09
+
+Description: Ingest data returned from Crowdstrike's Falcon Intelligence Actor APIs. A Crowdstrike Power-Up did not exist at the time of this release.
+
```

Comment:
May be worth adding a note that there should not be inbound nodes when using this snippet due to the use of `for` loops (could potentially be a bit of a node explosion depending on the row content)

Reply:
added